### DISCUSSION TITLE
linkerd-viz chart version rss

### DISCUSSION THREAD

**Discussion by jobinjosem1** [linkerd/linkerd2#10876] [Help]
Is there an rss feed url where I can get the latest stable chart version of linkerd-viz?
Foulnd this one https://github.com/linkerd/linkerd2/releases.atom but that only has info about the control plane and not viz

**Comment by alpeb** [comment]
You can check the artifacthub.io pages for linkerd-viz stable and edge:
https://artifacthub.io/packages/helm/linkerd2/linkerd-viz
https://artifacthub.io/packages/helm/linkerd2-edge/linkerd-viz
They contain an rss widget on the right sidebar.
